webserver/static/scripts/datasets/eval-jobs-viewer.js

```diff
@@ -1,3 +1,4 @@
+/** @jsx React.DOM */
```

Comment:
@gentlecat says that this is a requirement for older versions of react. I don't think we need it. @gentlecat can you confirm?